daemon/mgr/container.go

```diff
@@ -444,6 +444,14 @@ func (mgr *ContainerManager) start(ctx context.Context, c *Container, detachKeys
 		if err == nil {
 			return
 		}
+
+		// release the container resources(network and containerio)
+		err = mgr.releaseContainerResources(c)
```

Comment:
if failed before prepareContainerNetwork, will already occur error here. WDTY??

Reply:
Yes, releaseContainerResources will still work. It will check the existence of network.